solutions/src/test/java/ru/sberbank/school/task08/ManagerTest.java

```diff
@@ -0,0 +1,11 @@
+package ru.sberbank.school.task08;
+
+public abstract class ManagerTest {
+
+    protected static final String PATH =
+        "D:\\dev\\java\\Test";
```

Comment:
&#39;&quot;D:\\dev\\java\\Test&quot;&#39; has incorrect indentation level 8, expected level should be 12.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1534958748>*